src/PriceCalculatorKata/Utils/TaxCalculations.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace PriceCalculatorKata
+{
+    class TaxCalculation
+    {
+        private double _taxPercentage;
+        public double TaxPercentage { get { return _taxPercentage; } set { _taxPercentage = value.CastPercentage(); } }
+        public DiscountPrecedence UniversalDiscountPrecedence { get; set; } = DiscountPrecedence.AfterTax;
+        public DiscountPrecedence UPCDiscountPrecedence { get; set; } = DiscountPrecedence.AfterTax;
+        public Product Product { get; set; }
+        public double UPCDiscountAmount { get; set; }
+        public double UniversalDiscountAmount { get; set; }
+        public double GetTaxAmount()
```

Comment:
lets add new lines between methods and props. Just a coding convention ;)

Reply:
just a single new line between the methods

Reply:
done and pushed